.github/workflows/dev_test_optimum_common.yml

```diff
@@ -12,10 +12,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version:
-        - 3.7
-        - 3.8
-        - 3.9
+        python-version: ['3.9', '3.11']
```

Comment:
Why not 3.10?

Reply:
3.11 also has the security status : https://devguide.python.org/versions/ so should be tested